crates/provider-wit-bindgen/tests/bindgen_kv_memory.rs

```diff
@@ -0,0 +1,113 @@
+use std::process::Stdio;
+
+use anyhow::{Context, Result};
+use escargot::CargoBuild;
+use tempfile::TempDir;
+use tokio::process::Command;
+use uuid::Uuid;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_kv_memory() -> Result<()> {
+    let wasmcloud_bin_path =
+        std::env::var("TEST_WASMCLOUD_BIN").context("no wasmcloud bin provided")?;
```

Comment:
I strongly disagree with this, we already have logic to start the host as a library https://github.com/wasmCloud/wasmCloud/blob/28d2d6fc5e68ab8de12771fb3b0fb00617b32b30/tests/wasmbus.rs#L470-L484, there is no reason for tests to depend on a binary build of `wasmCloud`

Reply:
Yeah I'm aware of the `Host` & `HostConfig` code -- what I wanted to do was to test *the output* of `wasmcloud/wasmcloud` -- as you know people don't interact with `Host` -- they interact with the `wasmcloud` binary. 

The goal was for this test to test Bindgen worked (and continues to work) end to end.